<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BeagleMain</class>
 <widget class="QMainWindow" name="BeagleMain">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>729</width>
    <height>490</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>721</width>
    <height>460</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>BeagleTomb v.0.1.7</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../images/BTres.qrc">
    <normaloff>:/images/beagletomb_icon.png</normaloff>:/images/beagletomb_icon.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_11">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_10">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_11">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_9">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QComboBox" name="MODE_combo">
              <property name="minimumSize">
               <size>
                <width>121</width>
                <height>29</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>121</width>
                <height>29</height>
               </size>
              </property>
              <item>
               <property name="text">
                <string>Mode:</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Artists</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Albums</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Songs</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Videos</string>
               </property>
              </item>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="but_remote_tog">
              <property name="minimumSize">
               <size>
                <width>60</width>
                <height>29</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>60</width>
                <height>29</height>
               </size>
              </property>
              <property name="text">
               <string>Remote</string>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
              <property name="checked">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>200</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QListView" name="MenuList">
              <property name="minimumSize">
               <size>
                <width>200</width>
                <height>200</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QListView" name="TitleList">
              <property name="minimumSize">
               <size>
                <width>300</width>
                <height>200</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTabWidget" name="tabWidget">
          <property name="minimumSize">
           <size>
            <width>161</width>
            <height>200</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>300</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="toolTip">
           <string/>
          </property>
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="Playlist_tab">
           <attribute name="title">
            <string>Playlist</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_2">
              <item>
               <widget class="QListView" name="PlayList">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>170</width>
                  <height>200</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>300</width>
                  <height>800</height>
                 </size>
                </property>
                <property name="toolTip">
                 <string>loop/shuffle coming soon</string>
                </property>
               </widget>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_7">
                <property name="sizeConstraint">
                 <enum>QLayout::SetDefaultConstraint</enum>
                </property>
                <item>
                 <widget class="QPushButton" name="ADD_but">
                  <property name="minimumSize">
                   <size>
                    <width>55</width>
                    <height>30</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>55</width>
                    <height>30</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>Add</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="REMOVE_but">
                  <property name="minimumSize">
                   <size>
                    <width>55</width>
                    <height>30</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>55</width>
                    <height>30</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>Remove</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="DOWN_but">
                  <property name="minimumSize">
                   <size>
                    <width>24</width>
                    <height>30</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>55</width>
                    <height>30</height>
                   </size>
                  </property>
                  <property name="toolTip">
                   <string>Down</string>
                  </property>
                  <property name="text">
                   <string/>
                  </property>
                  <property name="icon">
                   <iconset resource="../images/BTres.qrc">
                    <normaloff>:/images/DOWN_icon.png</normaloff>:/images/DOWN_icon.png</iconset>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="UP_but">
                  <property name="minimumSize">
                   <size>
                    <width>24</width>
                    <height>30</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>55</width>
                    <height>30</height>
                   </size>
                  </property>
                  <property name="toolTip">
                   <string>Up</string>
                  </property>
                  <property name="text">
                   <string/>
                  </property>
                  <property name="icon">
                   <iconset resource="../images/BTres.qrc">
                    <normaloff>:/images/UP_icon.png</normaloff>:/images/UP_icon.png</iconset>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
           </layout>
           <zorder>PlayList</zorder>
           <zorder>layoutWidget</zorder>
           <zorder>verticalSpacer</zorder>
           <zorder>widget</zorder>
           <zorder></zorder>
           <zorder>layoutWidget</zorder>
          </widget>
          <widget class="QWidget" name="tab">
           <attribute name="title">
            <string>Radio</string>
           </attribute>
           <widget class="QWidget" name="layoutWidget">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>201</width>
              <height>341</height>
             </rect>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout">
             <item>
              <widget class="QListView" name="list_radio"/>
             </item>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_7">
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_6">
                 <item>
                  <widget class="QLabel" name="label_4">
                   <property name="text">
                    <string>Name</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLineEdit" name="entry_radName"/>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_5">
                 <item>
                  <widget class="QLabel" name="label_3">
                   <property name="text">
                    <string>URL     </string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLineEdit" name="entry_radURL"/>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_9">
                 <item>
                  <widget class="QPushButton" name="but_RadAdd">
                   <property name="minimumSize">
                    <size>
                     <width>80</width>
                     <height>27</height>
                    </size>
                   </property>
                   <property name="text">
                    <string>Add</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="but_RemRad">
                   <property name="minimumSize">
                    <size>
                     <width>80</width>
                     <height>27</height>
                    </size>
                   </property>
                   <property name="text">
                    <string>Remove</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </widget>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_8">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QLabel" name="SONG_lbl">
            <property name="minimumSize">
             <size>
              <width>300</width>
              <height>17</height>
             </size>
            </property>
            <property name="text">
             <string>Current Song</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_6">
            <item>
             <widget class="QLabel" name="SEEK_lbl">
              <property name="text">
               <string>00:00</string>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_5">
              <item>
               <widget class="QSlider" name="SEEK_slider">
                <property name="minimumSize">
                 <size>
                  <width>160</width>
                  <height>19</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>350</width>
                  <height>22</height>
                 </size>
                </property>
                <property name="toolTip">
                 <string>Seek</string>
                </property>
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
               </widget>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_3">
                <item>
                 <widget class="QPushButton" name="RRWD_but">
                  <property name="minimumSize">
                   <size>
                    <width>25</width>
                    <height>25</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>25</width>
                    <height>25</height>
                   </size>
                  </property>
                  <property name="toolTip">
                   <string>Prev</string>
                  </property>
                  <property name="text">
                   <string/>
                  </property>
                  <property name="icon">
                   <iconset resource="../images/BTres.qrc">
                    <normaloff>:/images/RRWD_icon.png</normaloff>:/images/RRWD_icon.png</iconset>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="STOP_but">
                  <property name="minimumSize">
                   <size>
                    <width>25</width>
                    <height>25</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>25</width>
                    <height>25</height>
                   </size>
                  </property>
                  <property name="toolTip">
                   <string>Stop</string>
                  </property>
                  <property name="text">
                   <string/>
                  </property>
                  <property name="icon">
                   <iconset resource="../images/BTres.qrc">
                    <normaloff>:/images/STOP_icon.png</normaloff>:/images/STOP_icon.png</iconset>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="PAUSE_but">
                  <property name="minimumSize">
                   <size>
                    <width>25</width>
                    <height>25</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>25</width>
                    <height>25</height>
                   </size>
                  </property>
                  <property name="toolTip">
                   <string>Pause</string>
                  </property>
                  <property name="text">
                   <string/>
                  </property>
                  <property name="icon">
                   <iconset resource="../images/BTres.qrc">
                    <normaloff>:/images/PAUSE_icon.png</normaloff>:/images/PAUSE_icon.png</iconset>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="FWD_but">
                  <property name="minimumSize">
                   <size>
                    <width>25</width>
                    <height>25</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>25</width>
                    <height>25</height>
                   </size>
                  </property>
                  <property name="toolTip">
                   <string>Play</string>
                  </property>
                  <property name="text">
                   <string/>
                  </property>
                  <property name="icon">
                   <iconset resource="../images/BTres.qrc">
                    <normaloff>:/images/FWD_icon.png</normaloff>:/images/FWD_icon.png</iconset>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="FFWD_but">
                  <property name="minimumSize">
                   <size>
                    <width>25</width>
                    <height>25</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>25</width>
                    <height>25</height>
                   </size>
                  </property>
                  <property name="toolTip">
                   <string>Next</string>
                  </property>
                  <property name="text">
                   <string/>
                  </property>
                  <property name="icon">
                   <iconset resource="../images/BTres.qrc">
                    <normaloff>:/images/FFWD_icon.png</normaloff>:/images/FFWD_icon.png</iconset>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="but_loop_single">
                  <property name="minimumSize">
                   <size>
                    <width>24</width>
                    <height>24</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>24</width>
                    <height>24</height>
                   </size>
                  </property>
                  <property name="toolTip">
                   <string>Repeat</string>
                  </property>
                  <property name="text">
                   <string/>
                  </property>
                  <property name="icon">
                   <iconset resource="../images/BTres.qrc">
                    <normaloff>:/images/RPT_icon.png</normaloff>:/images/RPT_icon.png</iconset>
                  </property>
                  <property name="checkable">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>208</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QDial" name="VOL_dial">
            <property name="minimumSize">
             <size>
              <width>50</width>
              <height>40</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>50</width>
              <height>64</height>
             </size>
            </property>
            <property name="toolTip">
             <string>Volume</string>
            </property>
            <property name="value">
             <number>50</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label">
            <property name="minimumSize">
             <size>
              <width>67</width>
              <height>17</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>67</width>
              <height>17</height>
             </size>
            </property>
            <property name="text">
             <string>Volume</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_8">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_10">
              <item>
               <widget class="QLabel" name="VOL_lbl">
                <property name="text">
                 <string>50</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_2">
                <property name="text">
                 <string>%</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>729</width>
     <height>27</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionPreferences_2"/>
    <addaction name="actionImport_Audio"/>
    <addaction name="actionImport_Video"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionDonate_2"/>
   </widget>
   <widget class="QMenu" name="menuMediatomb">
    <property name="title">
     <string>Mediatomb</string>
    </property>
    <addaction name="actionSync_2"/>
    <addaction name="actionAdmin"/>
   </widget>
   <widget class="QMenu" name="menuPlaylist">
    <property name="title">
     <string>Playlist</string>
    </property>
    <addaction name="actionSave"/>
    <addaction name="actionSave_2"/>
    <addaction name="actionOpen"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuMediatomb"/>
   <addaction name="menuPlaylist"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionDocumentation">
   <property name="text">
    <string>Documentation</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionAccess_Mediatomb">
   <property name="text">
    <string>Access Mediatomb</string>
   </property>
  </action>
  <action name="actionAdmin_Mediatomb">
   <property name="text">
    <string>Admin Mediatomb</string>
   </property>
  </action>
  <action name="actionPreferences">
   <property name="text">
    <string>Configuration</string>
   </property>
  </action>
  <action name="actionPreferences_2">
   <property name="text">
    <string>Preferences</string>
   </property>
  </action>
  <action name="actionDonate">
   <property name="text">
    <string>Donate</string>
   </property>
  </action>
  <action name="actionSync">
   <property name="text">
    <string>Sync</string>
   </property>
  </action>
  <action name="actionDonate_2">
   <property name="text">
    <string>Donate</string>
   </property>
  </action>
  <action name="actionAdmin">
   <property name="text">
    <string>Admin</string>
   </property>
  </action>
  <action name="actionSync_2">
   <property name="text">
    <string>Sync</string>
   </property>
  </action>
  <action name="actionImport_Audio">
   <property name="text">
    <string>Import Audio</string>
   </property>
  </action>
  <action name="actionImport_Video">
   <property name="text">
    <string>Import Video</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionSave_2">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../images/BTres.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>SEEK_slider</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>SEEK_lbl</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>365</x>
     <y>355</y>
    </hint>
    <hint type="destinationlabel">
     <x>423</x>
     <y>316</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>VOL_dial</sender>
   <signal>valueChanged(int)</signal>
   <receiver>VOL_lbl</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>567</x>
     <y>350</y>
    </hint>
    <hint type="destinationlabel">
     <x>676</x>
     <y>353</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
